presto-native-execution/presto_cpp/presto_protocol/connector/iceberg/presto_protocol-json-hpp.mustache

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
```

Comment:
@abhiseksaikia : Can we code-gen these files ? There is lots of repetition of mustache templates.

Reply:
Thank you for reviewing the change. I was planning to address this comment, along with Andrii's comment above, in a follow-up PR. Please let me know if you have any concerns.